lib/active_record/open_tracing/processor.rb

```diff
@@ -45,12 +48,16 @@ def tags_for_payload(payload)
           "span.kind" => SPAN_KIND,
           "db.instance" => db_instance,
           "db.cached" => payload.fetch(:cached, false),
-          "db.statement" => payload.fetch(:sql).squish,
+          "db.statement" => sanitize_sql(payload.fetch(:sql).squish),
           "db.type" => DB_TYPE,
           "peer.address" => db_address
         }
       end
 
+      def sanitize_sql(sql)
+        sanitizer ? sanitizer.sanitize(sql) : sql
```

Comment:
should this return empty string if the sanitizer is not defined?

Reply:
That's probably a safer default behavior. I will change that.